medallion/__init__.py

```diff
@@ -92,3 +107,37 @@ def handle_error(error):
     return Response(response=flask.json.dumps(error),
                     status=500,
                     mimetype=MEDIA_TYPE_TAXII_V20)
+
+
+@application_instance.errorhandler(ProcessingError)
+def handle_processing_error(error):
+    _modify_py2_exception(error)  # python 2 --> python3 exception format
+
+    e = {
+        "title": "ProcessingError",
+        "http_status": "422",
+        "exception": str(error.args[0])
```

Comment:
We should  use `description` rather than `exception`, to match the [TAXII 2.0 `error` type](http://docs.oasis-open.org/cti/taxii/v2.0/cs01/taxii-v2.0-cs01.html#_Toc496542721). Also, rather than handling `args`, it might be better to just define a `__str__` representation of the exceptions and use that. Otherwise, we could put more details into the `details` attribute of the error, but I wouldn't worry about that until we have much more structured errors.

Reply:
Roger.